src/main/java/core/basesyntax/storage/impl/StorageEntry.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.storage.impl;
+
+public class StorageEntry {
+    private String product;
+    private int quantity;
+
+    public StorageEntry(String product, int quantity) {
+        this.product = product;
+        this.quantity = quantity;
+    }
+
+    public String getProduct() {
+        return product;
+    }
+
+    public void setProduct(String product) {
+        this.product = product;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+}
```

Comment:
Consider encapsulating the `StorageEntry` class. Since `StorageEntry` is a part of the `storage.impl` package, it's likely that it should be used internally within the storage package. Therefore, it would be better to restrict its visibility to package-private if it's not intended to be used outside the storage package. This would follow the encapsulation principle by not exposing the class unnecessarily.